### ISSUE TITLE
Commander misplaced structures

### ISSUE THREAD

**Issue by jirikivaari** [ENSL/NS#44]
Sometimes as a commander you drop structures in a different position that is some static coordinates to left and below. Happens for medpacks too.

**Comment by brcaswell** [comment]
sure it wasn't as intended 😃 

seriously, we probably need version build info, which client, what resolution and more clarity on 'sometimes', is it reproducible or have you noticed any other attributing factors to when this sometimes occurs?

**Comment by jirikivaari** [comment]
Yes I have no idea what is causing this. I can get the client and resolution next time I get this. It doesn't happen that often. Latest client. 3.2.2.

It happens from the start of commanding. And I think it has something to do with changes that fmoraw, Bacsu. etc. made to make this Linux-compatible, it came with the other 3.2.2 bugs.

Basically there is some fixed offset, and you have to log out to fix it.

**Comment by jirikivaari** [comment]
At least Fairy has had this bug.

**Comment by jirikivaari** [comment]
Still happens with latest client (3.2.2) and server version same.

Did manage to have it after but these are not repro steps.

- Enhanced pointer precision turned on in windows
- m_rawinput 0
- NOT using --noforceparams

Also computer is Windows 10 Pro 64-bit, Intel i5-7600K, NVIDIA GeForce GTX 1060 6GB, 2560 x 1440 (32 bit) (144Hz), 

Map was ns_metal, but I've had it in ns_veil and ns_tanith too. 

It could be related to SDL, mouse input or possibly the command view distance calculus.
